src/test/java/com/origamisoftware/weekone/AppTest.java

```diff
@@ -16,4 +16,10 @@ public void testMessage() {
         assertEquals("Verify the message is set correctly", message, app.getMessage());
     }
 
+    @Test(expected = IllegalArgumentException.class)
+    public void testNegative() {
+        String message = "Hello World";
+        App app = new App(null);
+        assertEquals("Verify the message is set correctly", message, app.getMessage());
```

Comment:
use assert null here. 
this is better:
 @Test(expected = IllegalArgumentException.class)
    public void testNegative() {
        App app = new App(null);
        assertNull("Verify the message is set correctly", app.getMessage());
    }